### PR TITLE
Dev sqlserver

### PR DESCRIPTION
**Pull request by joeaero19**

### CODE REVIEW

File: data/aws/postgres.tf [hunk 1]
```diff
@@ -69,7 +69,8 @@ resource "aws_security_group_rule" "vpn" {
 }
 
 resource "aws_db_parameter_group" "db_param_group" {
-  name   = "rds-pg-${terraform.workspace}"
+  name   = "rds-postgres-${terraform.workspace}"
```
Comment (dwhitecl) [inline data/aws/postgres.tf:72]: why is this line changed @joeaero19 ?

**Review by dwhitestratiform — COMMENTED** [review]
please check one comment in the PR otherwise it looks good to me

File: data/aws/postgres.tf (hunk 1 above)
Comment (joeaero19) [reply data/aws/postgres.tf:72]: It was complaining about the name when I was testing. Ignore this I am correcting it now.

**Comment by dwhitecl** [comment]
no issues from me. waiting for @mdial89f  to review as well

**Review by mdial89f — CHANGES_REQUESTED** [review]
Remove all instances of api sqlserver from the docker-compose.yml and coker-compose.dev.yml

**Review by mdial89f — APPROVED** [review]
